utils/MonitoredResource.md

```diff
@@ -0,0 +1,83 @@
+# Monitored Resource
+
+This document is about the monitored resource, what resources are supported and how to automatically
+detect these resources.
+
+## Supported Monitored Resources
```

Comment:
Should we also mention the global monitored resources? If user's binary is not running in any of these environments, a default global resource will be used.

Reply:
I don't think we need that here. That is specific to SD monitoring, if we run in an unknown environment we can return `null` monitored resource.

Reply:
SGTM